modules/core/src/test/scala/fs2/kafka/KafkaConsumerSpec.scala

```diff
@@ -540,6 +541,91 @@ final class KafkaConsumerSpec extends BaseKafkaSpec {
       }
     }
 
+    it("should handle rebalance with CooperativeStickyAssignor") {
```

Comment:
This is a copy of the `"should handle rebalance"` test, adapted to use the custom assignor, along with the resulting assert.